uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-assignments/services/document.js

```diff
@@ -17,10 +17,40 @@ define([
     });
   }]);
 
-  services.factory('DocumentService', ['Document', '$q', 'config', 'UtilsService', '$log',
-    function (Document, $q, config, UtilsService, $log) {
+  services.factory('DocumentService', ['Document', '$q', 'config', 'settings', 'UtilsService', 'ContactService', 'AssignmentService', '$log',
+    function (Document, $q, config, settings, UtilsService, ContactService, AssignmentService, $log) {
       $log.debug('Service: DocumentService');
 
+      function collectContactIds (documentList) {
```

Comment:
move private functions below the `return`, hide the implementation details at the bottom